Sponsors page

postalservice14 opened the pull request.

postalservice14 committed:
First draft sponsors page

jeremeamia commented:
thanks, I'll check it out.

postalservice14 commented on package.json:
```diff
@@ -21,6 +21,7 @@
     "gulp-size": "^1.2.1",
-    "gulp-uglify": "^1.1.0"
+    "gulp-uglify": "^1.1.0",
+    "vinyl-source-stream": "^1.1.0"
```
I had to add this to get mine to build...not sure if it was just on my machine or what...

jeremeamia commented on resources/views/layout.blade.php:
```diff
@@ -37,6 +37,7 @@
                 <li><a href="./speakers">Speakers</a></li>
                 <li><a href="./talks">Talks</a></li>
                 <li><a href="./events">Events</a></li>
+                <li><a href="./sponsors">Sponsors</a></li>
```
There is a sponsors link on the right side of the navbar. Don't need this one.

postalservice14 replied:
Ah true, didn't see that one

jeremeamia commented:
Looks good to me. I think we should still have the community sponsors tiled at the bottom like the other pages. Will have to do some jiggery pokery to get that in there without duplicating that content.

postalservice14 committed:
Added community sponsors to sponsors page.
Removed duplicate sponsors link
Split out community sponsors into blade section

postalservice14 commented:
"jiggery pokery" - done :-p

postalservice14 commented:
Addresses issue: https://github.com/pnwphp/pnwphp.com/issues/9

jeremeamia commented:
Awesome. Can you change your pull request to merge into the `develop` branch instead of `master`?
